src/functions.js

```diff
@@ -1,12 +1,43 @@
-const addScore = () => {
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+const addNewGame = (endpoint = '', gameName = '') => {
+  const targetUrl = url + endpoint;
+  const data = { name: gameName };
+  return fetch(targetUrl, {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  }).then((resp) => resp.json());
+};
+
+const getScore = (id = '') => {
+  const targetUrl = `${url}games/${id}/scores/`;
+  return fetch(targetUrl).then((resp) => resp.json());
+};
+
+const addScore = (user, score) => {
   const newSCore = document.createElement('div');
   const playersScores = document.querySelector('.players-scores');
   const newScoreOrder = playersScores.children.length + 1;
-  const [playerName, playerScore] = Array.from(document.querySelectorAll('.user-input'));
-  newSCore.innerHTML = `<p>${playerName.value}: ${playerScore.value}`;
+  newSCore.innerHTML = `<p>${user}: ${score}`;
   newSCore.className = 'score';
   if (newScoreOrder % 2 === 0) newSCore.style.backgroundColor = 'rgb(221,221,221)';
   else newSCore.style.backgroundColor = '#fff';
   playersScores.appendChild(newSCore);
 };
-export default addScore;
\ No newline at end of file
+const postScore = (userName, userScore, id) => {
+  const targetUrl = `${url}games/${id}/scores`;
+  const data = { user: userName, score: userScore };
+  return fetch(targetUrl, {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  }).then((resp) => resp.json());
```

Comment:
- [ ] <b>REQUIRED:</b> As requested in project requirements, you have to use `async/await` in case of any API request in this project. Kindly do that to comply with the project specifications.